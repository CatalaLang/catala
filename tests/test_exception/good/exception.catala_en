## Test

```catala
declaration scope A:
  context x content integer

scope A:
  label base_x
  definition x equals 0

  exception base_x
  definition x equals 1
```
