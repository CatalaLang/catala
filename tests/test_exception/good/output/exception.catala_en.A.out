[RESULT] Computation successful! Results:
[RESULT] x = 1
